monitor/grafana/zeebe.json

```diff
@@ -1358,14 +1356,14 @@
               "refId": "A"
             },
             {
-              "expr": "max(kube_pod_container_resource_limits_memory_bytes{namespace=~\"$namespace\",pod=~\".*zeebe.*\", container=~\"zeebe.*\"})",
+              "expr": "max(kube_pod_container_resource_limits_memory_bytes{namespace=~\"$namespace\",pod=~\"$pod\", container=~\"zeebe.*\"})",
               "hide": false,
               "interval": "",
               "legendFormat": "limit",
               "refId": "B"
             },
             {
-              "expr": "min(kube_pod_container_resource_requests_memory_bytes{namespace=~\"$namespace\",pod=~\".*zeebe.*\", container=~\"zeebe.*\"})",
+              "expr": "min(kube_pod_container_resource_requests_memory_bytes{namespace=~\"$namespace\",pod=~\"$pod\", container=~\"zeebe.*\"})",
```

Comment:
Could you try this on the camunda cloud int stage as well, please. I can remember there were some issues around that but idk what was it, might be the value of the pod variable or something. Please try it out if this works than we can merge it :)

Reply:
Tested on https://grafana-worker-1.internal.camunda.io/ where I did not see a difference, except that pod can now be filtered on.